.github/workflows/coverage.yml

```diff
@@ -25,10 +29,15 @@ jobs:
 
       - name: Install Dependencies
```

Comment:
**Suggestion:** In a last effort to figure out why this isn't working I think we should try:

- fetch trunk
- check out trunk
- install node and deps in trunk
- run jest coverage report on trunk and write it to a shared higher directory 
- checkout ${{ github.event.pull_request.head.sha }}
- install node and deps in `${{ github.event.pull_request.head.sha }}`
- run jest coverage report on `${{ github.event.pull_request.head.sha }}` branch and write it to a shared higher directory 
- call the prebuilt action and pass the paths to the two hopefully-existing test reports https://github.com/ArtiomTr/jest-coverage-report-action#use-existing-test-reports

add logs liberally to try to figure out the point of failure, etc.

Also please try using the checkout action consistently for both checkouts:

https://github.com/actions/checkout#checkout-a-different-branch